uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerMicrosoftAuthenticator.java

```diff
@@ -198,21 +198,21 @@ public void enableBrowserAccess() {
     }
 
     @Override
-    public void createPowerLiftIncident() {
+    public String createPowerLiftIncident() {
         Logger.i(TAG, "Creating Power Lift Incident..");
         launch();
         if (shouldHandleFirstRun) {
             handleFirstRun();
         }
 
         if (isInSharedDeviceMode) {
```

Comment:
What can change this flag at runtime?  Anything?

Reply:
Successful completion of shared device registration sets this to true here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerMicrosoftAuthenticator.java#L150